test/functional/notes_controller_test.rb

```diff
@@ -1061,4 +1061,21 @@ def test_get_rss_feed
      assert_equal I18n.t('notes_controller.saved_as_draft'), flash[:notice]
      assert_redirected_to '/notes/' + users(:jeff).username + '/' + Time.now.strftime('%m-%d-%Y') + '/' + title.parameterize
    end
+
+  # Preview test
```

Comment:
Is the test not working? Seems correct 👍

Reply:
I was using it to test the route, but it's not passing yet because I discovered that the node is actually being created..
I have updated the screenshot 

https://github.com/publiclab/plots2/pull/8620#issuecomment-713725564

Reply:
It is not getting saved for me:

<img width="1258" alt="Screenshot 2020-10-22 at 4 12 42 PM" src="https://user-images.githubusercontent.com/53554917/96861299-83d45700-1481-11eb-8cd0-c492bd4392ac.png">

Reply:
Interesting.. I'll retry it